src/components/table/table-cell/table-cell.spec.js

```diff
@@ -122,11 +123,19 @@ describe('TableCell', () => {
         });
 
         it('renders a textarea that matches the expected style', () => {
-          wrapper = mount(
+          const wrapper1 = mount(
```

Comment:
why do we need another wrapper?

Reply:
I need to run two tests to meet coverage, I'll remove the extra wrapper and just reassign it the new component